sdcm/nemesis.py

```diff
@@ -1846,7 +1848,9 @@ def disrupt_truncate_large_partition(self):
         # In order to workaround issue #4924 when truncate timeouts, we try to flush before truncate.
         self.target_node.run_nodetool("flush")
         # do the actual truncation
-        self.target_node.run_cqlsh(cmd='TRUNCATE {}.{}'.format(ks_name, table), timeout=120)
+        truncate_timeout = 600
+        self.target_node.run_cqlsh(cmd=f'TRUNCATE {ks_name}.{table} USING TIMEOUT {int(truncate_timeout)}s',
+                                   timeout=120)
```

Comment:
Shouldn't the `timeout=120` be changed too?

Reply:
yes, you are right, i forgot to change it